src/foam/nanos/crunch/MinMaxCapabilityData.js

```diff
@@ -38,7 +38,7 @@ foam.CLASS({
           ];
         }
 
-        if ( foam.Function.isInstance(n.toString) ) {
+        if ( foam.Function.isInstance(n.toString()) ) {
```

Comment:
When does this happen? At least add a comment to negate the weirdness if it is correct.